bin/check-services

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+"""
+Exit with 0 if all the given services are healthy, 1 otherwise.
+
+Usage:
+
+    bin/check-services lms_postgres_1 ...
+
+"""
+from sys import argv, exit
+from subprocess import check_output, DEVNULL
+
+
+def check_service(service):
+    try:
+        status = (
+            check_output(
+                [
+                    "docker",
+                    "inspect",
+                    "-f",
+                    "{{.State.Health.Status}}",
+                    service,
+                ],
+                stderr=DEVNULL,
+            )
```

Comment:
This `docker inspect` command can be used to check the `healthcheck` of a container:

```terminal
$ docker inspect -f {{.State.Health.Status}} lms_postgres_1
health
```